### ISSUE TITLE
In Animatable.prototype._animate - sometimes the variable _previousRatio and _ratioOffset contains NaN

### ISSUE THREAD

**Issue by hedgehed** [BabylonJS/Babylon.js#4203]
I had to add validation and correction to these variables for improve stability.

```

Animatable.prototype._animate = function (delay) {
.........
........
     for (index = 0; index < runtimeAnimations.length; index++) {
            	var animation = runtimeAnimations[index];

            	// this added
		if(isNaN(animation._previousRatio))
            	{
            		animation._previousRatio = 0;
            	}
		
            	//and this
            	if(isNaN(animation._ratioOffset))
            	{
            		animation._ratioOffset = 0;
            	}

                var isRunning = animation.animate(delay - this._localDelayOffset, this.fromFrame, this.toFrame, this.loopAnimation, this._speedRatio);
................


```

I can not show, how to reproduce the bug, but this patch helped me

**Comment by sebavan** [comment]
This looks more like a workaround than a fix I would really like to understand the conditions creating the NaN and fix the root cause/

Is it really not possible to repro in the playground ?

**Comment by hedgehed** [comment]
In the playground I can not reach the my bug because animation loop problem:
http://www.babylonjs-playground.com/indexstable#JSI21K
When i try to change "true" to "false"  this line 
`scene.beginAnimation(skeletons2[0], 2, 32, true, 1.0);`
Animation is not stops, but on the contrary plays all frames nonstop

Model exported from 3dsmax, 
"exporter_version": "0.4.5",

**Comment by deltakosh** [comment]
It is because your file is in automatic start mode for all animations.

Here is your fix:
http://www.babylonjs-playground.com/index.html#JSI21K#1

TL;DR: Just call scene.stopAllAnimations() to stop animations started by the file
